app/services/order_details/param_updater.rb

```diff
@@ -58,7 +59,8 @@ def update_attributes(params)
 
     user_newly_assigned = @order_detail.assigned_user_id_changed? && @order_detail.assigned_user.present?
 
-    @order_detail.manually_priced!
+    @order_detail.manually_priced! # this seems like a misleading method name
```

Comment:
I'm not opposed to renaming this. This is probably a good place to update set the price_changed_by_user, though this method is now getting pretty big.

One thing we'll want to take into account is that if a user manually changes the price, then another user comes and changes something else (like the note), it should not overwrite the price changed by user.